tool_collections/kraken/kraken_report/kraken-mpa-report.xml

```diff
@@ -10,14 +10,24 @@
     <command>
 <![CDATA[
 
+#set $names = []
+
 #for $input_count, $input_classification in enumerate( $classification ):
-    ln -s "${input_classification}" "sample_${input_count}" &&
+    #set $name_base = str( getattr( $input_classification, 'element_identifier', 'sample' ) ).replace( "/", '-' ).replace( "\t", "-" )
```

Comment:
Element identifier is populated for all dataset wrappers - is this to explicitly enable like really old Galaxies or is there some edge case I am missing?